staging/src/k8s.io/kube-proxy/config/v1alpha1/types.go

```diff
@@ -17,8 +17,8 @@ limitations under the License.
 package v1alpha1
 
 import (
-	apimachineryconfigv1alpha1 "k8s.io/apimachinery/pkg/apis/config/v1alpha1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	apimachineryconfigv1alpha1 "k8s.io/component-base/config/v1alpha1"
```

Comment:
it appears that while the import path was updated, the name was left the same. This should be `componentbaseconfigv1alpha1 "k8s.io/component-base/config/v1alpha1"`

Reply:
Good catch!